src/test/java/org/testeditor/fixture/host/integrationtest/HostDriverIntegrationTest.java

```diff
@@ -0,0 +1,27 @@
+package org.testeditor.fixture.host.integrationtest;
+
+import org.testeditor.fixture.host.HostDriverFixture;
+
+import org.junit.Assert;
+
+public class HostDriverIntegrationTest {
+
+    // @Test
```

Comment:
`@Test` is ok but you should add `@Ignore` instead - e.g. `@Ignore("manual execution only")`